elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/impl/ElasticsearchIndexManager.java

```diff
@@ -100,6 +127,11 @@ private IndexManagementStrategy getIndexManagementStrategy(Properties properties
 		return strategy != null ? IndexManagementStrategy.valueOf( strategy ) : IndexManagementStrategy.NONE;
 	}
 
+	private String getIndexManagementWaitTimeout(Properties properties) {
+		String timeout = properties.getProperty( ElasticsearchEnvironment.INDEX_MANAGEMENT_WAIT_TIMEOUT );
+		return timeout != null ? Integer.valueOf( timeout ) + "ms" : DEFAULT_INDEX_MANAGEMENT_WAIT_TIMEOUT;
```

Comment:
Do we need better validation, i.e. being a positive number and a reasonable range?